src/inmanta/plugins.py

```diff
@@ -216,7 +216,7 @@ def type_string_internal(self) -> str:
     "any": inmanta_type.Type(),  # Any value will pass validation
     "expression": inmanta_type.Type(),  # Any value will pass validation
     "null": Null(),  # Only NoneValue will pass validation
-    None: Null(),  # Only NoneValue will pass validation
+    None: inmanta_type.Type(),  # In iso6, a None return type accepts any value, from iso7+ it expects a None value.
```

Comment:
This is the change to make iso6 backward compatible with regard to returned types validation